esm_runscripts/prepare.py

```diff
@@ -442,7 +442,7 @@ def _add_all_folders(config):
         "config",
         "restart_in",
     ]
-    config["general"]["reusable_filetypes"] = ["bin", "src"]
+    config["general"]["reusable_filetypes"] = config["general"].get("reusable_filetypes", ["bin", "src"])
```

Comment:
Looks good. That means, it is taken either from the user runscript, or from the setup yaml, right?

Reply:
Correct. For example in awicm3.yaml:
https://github.com/esm-tools/esm_tools/blob/feature/awicm3/configs/setups/awicm3/awicm3.yaml#L40